build/rive_build_config.lua

```diff
@@ -422,7 +435,7 @@ if _OPTIONS['for_android'] then
 
     -- Detect the NDK.
     local EXPECTED_NDK_VERSION = 'r27c'
-    local NDK_LONG_VERSION_STRING = "27.2.12479018"
+    local NDK_LONG_VERSION_STRING = "25.2.9519653"
```

Comment:
Thanks for calling attention to this! @ErikUggeldahl can probably suggest how we should handle it.

Reply:
Yeah, this entire PR is highlighting customer hot spots. So, what we likely need is a good way to specify the NDK/SDK version. I'm not sure what is the best way to do it, but basically something like this is a common approach:
```
export ANDROID_SDK_ROOT=/path/to/android/sdk
export ANDROID_NDK_ROOT=/path/to/android/ndk
export ANDROID_ABI=arm64-v8a
export ANDROID_PLATFORM=android-21
export ANDROID_TOOLCHAIN=clang
export ANDROID_STL=c++_shared
```
And then you will of course have your own defaults.